doc/content/integrations/payload-formatters/grpc.md

```diff
@@ -0,0 +1,7 @@
+---
+title: "GRPC"
+description: ""
+draft: true
+---
+
+GRPC Payload Formatters are not yet implemented.
```

Comment:
I guess we should display a notification in the console to warn the users that this option is not currently supported, or to hide this option.

@johanstokking ?

Reply:
I wouldn't even mention this in the docs